test/kitchen/test/integration/system-probe-test/rspec/system-probe-test_spec.rb

```diff
@@ -20,6 +20,32 @@
   "pkg/collector/corechecks/ebpf/probe"
 )
 
+TIMEOUTS = {
+  "pkg/network/protocols" => "5m",
+  # disable timeouts for pkg/network/tracer
+  "pkg/network/tracer$" => "0",
+}
+
+DEFAULT_TIMEOUT = "10m"
+
+def get_timeout(package)
+  match_size = 0
+  timeout = DEFAULT_TIMEOUT
+
+  # determine longest match
+  TIMEOUTS.each do |k, v|
+    k = Regexp.new(k)
```

Comment:
nit: you could probably move `Regexp.new` into the dictionary definition, so it is only done once.